g4f/Provider/deprecated/Yqcloud.py

```diff
@@ -1,10 +1,10 @@
 from __future__ import annotations
 
 import random
-from ..requests import StreamSession
+from ...requests import StreamSession
```

Comment:
Consider adding an import statement for 'StreamSession' to the imports section at the beginning of the file.